pom.xml

```diff
@@ -351,6 +357,13 @@
             <version>1.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>graal-sdk</artifactId>
+            <version>21.3.11</version>          <!-- The latest version supports JDK8 -->
```

Comment:
at first glance, I thought you were saying 21.3.11 is the latest version, and we chose it because it supports JDK8
```suggestion
            <version>21.3.11</version> <!-- 21.3.11 is the last version to support JDK8 -->
```